login-workflow/src/new-architecture/screens/ResetPasswordScreen/ResetPasswordScreen.tsx

```diff
@@ -66,13 +81,23 @@ export const ResetPasswordScreen: React.FC<ResetPasswordScreenProps> = (props) =
         [setPasswordInput, setConfirmInput]
     );
 
+    const errorConfig: ErrorManagerProps = {
+        mode: 'dialog',
+        error: errorMsg,
+        dialogConfig: {
+            title: t('bluiCommon:MESSAGES.ERROR'),
+            dismissLabel: t('bluiCommon:ACTIONS.OKAY'),
+        },
+        onClose: (): void => { setErrorMsg(''); navigate(routeConfig.LOGIN)},
+    };
```

Comment:
Most of these are defaults. The only thing that isn;t is the error message and the on close method. The error message should be handled with the `triggerError` hook from the errormanager.